<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="Menu">
     <property name="styleSheet">
      <string notr="true">#Menu{background-image: url(:/res/ice-bck.png); }
</string>
     </property>
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>411</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/doodle-jump.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/play.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pic2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="covertail">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>700</y>
        <width>600</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/cover.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>150</y>
        <width>131</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/res/UFO.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/res/platform3.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="choose">
     <property name="styleSheet">
      <string notr="true">#choose{
	background-image: url(:/res/ice-bck.png);
}</string>
     </property>
     <widget class="QLabel" name="chooselb">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>521</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Choose Theme</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="JungleBT">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>200</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Jungle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SpaceBt">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>370</y>
        <width>200</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Space</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UWBt">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>490</y>
        <width>200</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Underwater</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HopBT">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>600</y>
        <width>200</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Easter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">#page2{
      color: rgb(92, 92, 92);
}</string>
     </property>
     <widget class="QLabel" name="scoreLb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>600</width>
        <height>800</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pauselb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>35</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Resumebt">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>490</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/resume.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Pausebt">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/暫停.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pic1">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>151</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/res/UFO.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Hop_bottom">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>720</y>
        <width>600</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/hop-bottom (2).png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">#page_3 {border-image: url(:/res/jungle-bck.png);}</string>
     </property>
     <widget class="QPushButton" name="menuButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>510</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/res/menu.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="oberlabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>481</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>45</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 23, 73);</string>
      </property>
      <property name="text">
       <string>Game Over !</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="fscorelabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>551</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>35</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Final Score :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
